tests/components/mqtt/test_light_json.py

```diff
@@ -814,10 +885,11 @@ def __eq__(self, other):
 
     state = hass.states.get("light.test")
     assert state.state == STATE_OFF
-    assert state.attributes.get(ATTR_SUPPORTED_FEATURES) == 185
+    assert state.attributes.get(ATTR_SUPPORTED_FEATURES) == 187
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5199018)